doc/src/manual/modules.md

```diff
@@ -589,6 +593,12 @@ A few other points to be aware of:
 It is sometimes helpful during module development to turn off incremental precompilation. The
 command line flag `--compiled-modules={yes|no}` enables you to toggle module precompilation on and
 off. When Julia is started with `--compiled-modules=no` the serialized modules in the compile cache
-are ignored when loading modules and module dependencies. `Base.compilecache` can still be called
+are ignored when loading modules and module dependencies.
+More fine-grained control is available with `--pkgimage-native-code=no`, which suppresses only
+native-code storage during precompilation. `Base.compilecache` can still be called
 manually. The state of this command line flag is passed to `Pkg.build` to disable automatic
 precompilation triggering when installing, updating, and explicitly building packages.
+
+You can also debug some precompilation failures with environment variables. Setting
+`JULIA_VERBOSE_LINKING=true` may help resolve failures in linking shared libraries of compiled
+native code. See the `devdocs` section of the manual for further details.
```

Comment:
Could this be more specific to where in the devdocs this refers to?

Reply:
Turns out we can't link to devdocs from the main manual (I did it that way originally and CI failed).